.github/workflows/publish_to_pypi.yml

```diff
@@ -0,0 +1,116 @@
+name: Publish xrpl-py 🐍 distribution 📦 to PyPI
+on:
+  push:
+    tags:
+        - 'v*' # Run on version tags only
+
+jobs:
+  build:
+    name: Build distribution 📦
+    runs-on: ubuntu-latest
+    env:
+        POETRY_VERSION: 1.8.3
```

Comment:
Did you document this and other places where we use poetry to give contributors and understanding where all they need to make changes

Reply:
Hmm, I have not explicitly documented the usage of `poetry`. At the moment, it is used in all the Github Actions workflows. 

In another PR, the version of poetry has already been updated to `1.8.3` in all the Github Actions workflows. I need to rebase this feature branch.

Are you referring something else for documentation?